qark/templates/xml_report.jinja

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+{% for issue in issues %}
+<issue issueid="{{ issue.id }}">
```

Comment:
I don't think this attribute exists. What are we planning on putting here?

Also, what about the other attributes that `Vulnerability` has?

```category, issue_name, severity, description, line_number, file_object```

Reply:
The `id` attribute doesn't yet exist. The thinking goes that since we have one Report class now we can assign unique IDs to each issue.

As I mentioned in the PR description, the contents of the reports and the Vulnerability class are not finalized yet so these have been left only partially implemented.